core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -258,21 +259,18 @@ public T responseCauseSanitizer(Function<? super Throwable, ?> responseCauseSani
     }
 
     /**
-     * Sets the rate at which to sample requests to log. Any number between {@code 0.0} and {@code 1.0} will
-     * cause a random sample of the requests to be logged. The random sampling is appropriate for low-traffic
-     * (ex servers that each receive &lt;100K requests). If unset, all requests will be logged.
+     * Sets the {@link Sampler} that to sample requests to log.
      */
-    public T samplingRate(float samplingRate) {
```

Comment:
I think we can still keep this method, because random sampling is most popular and we should avoid braking backward compatibility:

```java
public T samplingRate(float samplingRate) {
    checkArgument(...)
    return sampler(Sampler.random(samplingRate));
}
```

Reply:
Thank you for review. Updated in 307406a.